production/loki-mixin/dashboards/dashboard-loki-operational.json

```diff
@@ -3926,19 +3926,19 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "histogram_quantile(.99, sum(rate(cortex_memcache_request_duration_seconds_bucket{cluster=\"$cluster\", namespace=\"$namespace\"}[5m])) by (method, name, le, container))",
+              "expr": "histogram_quantile(.99, sum(rate(loki_memcache_request_duration_seconds_bucket{cluster=\"$cluster\", namespace=\"$namespace\"}[5m])) by (method, name, le, container))",
```

Comment:
Hhmm, do you know when this metric name changed?
We may need to `or` it for folks who are running a version older than when the change was introduced.

Reply:
After discussing with @owen-d, we don't need to do this since the mixin is co-located with the version of Loki being run